waltz-ng/client/physical-flows/physical-flow-registration.js

```diff
@@ -93,7 +119,12 @@ function controller(
     const viewState = kindToViewState(sourceEntityRef.kind);
     vm.cancelLink = $state.href(viewState, { id: sourceEntityRef.id });
 
-    const doValidation = () => validate(vm.specification, vm.flowAttributes, vm.targetEntity);
+    const doValidation = () => {
+        vm.similarFlows = [];
+        vm.visibility.similarFlows = false;
```

Comment:
this is so we can have the 'show' button controlling whether or not those flows are visible.